.github/workflows/test.yml

```diff
@@ -71,7 +110,8 @@ jobs:
         with: 
           github-token: ${{ secrets.GITHUB_TOKEN }}
           sonar-token: ${{ secrets.SONAR_TOKEN }}
-          restore-artifact-name1: "test-report-files"
+          restore-artifact-name1: "test-report-files-transform"
```

Comment:
This adds the test info and coverage to Sonar, right, but if you see the artifacts, the aggregate test report is missing: 
![image](https://github.com/giis-uniovi/modevo/assets/10879637/a45e5760-a26b-4460-a628-e1cdece336be)

Reply:
I think that I added the aggregate test report now, is it correct and in the right place?